.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs/*.log
```

Comment:
could be a situation when we have `.txt` or maybe `.csv` logs, but that is not very important